sci-astronomy/libthesky/libthesky-0.4.1.ebuild

```diff
@@ -20,6 +20,8 @@ IUSE="static-libs"
 DEPEND="~sci-libs/libsufr-0.7.5[static-libs?]"
 RDEPEND="${DEPEND}"
 
+CMAKE_MAKEFILE_GENERATOR="emake"
+
```

Comment:
Could you move this to just below the `inherit cmake...` at the top of the file?